.gitignore

```
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+/logs/app.log
```

Let's ignore all folder.
```suggestion
/logs/*
```